index.bs

```diff
@@ -14,4 +14,63 @@ Group: WebView-CG
 
 # Introduction # {#introduction}
 
-Hello world!
\ No newline at end of file
+Hello world!
+
+# Motivating Applications # {#motivating-applications}
+
+*This section is non-normative.*
+
+## Latency-sensitive applications in virtualized sessions ## {#latency-sensitive-apps}
```

Comment:
I would like for us to refer to this case with the tag "#browser-content-redirection". Can you make that change?